src/user/configure-metadata.md

```diff
@@ -1821,6 +1824,49 @@ SubExpression notes:
 
 4. If you wish to evaluate an expression before aggregating that involves other types of data such as program data, or that involves more than one data element, category option combination or attribute option combination, you can use a Predictor to do so and store the result as a different data element. Then you can reference the predicted data element in an indicator or directly in analytics.
 
+### Indicator Year-to-date { #indicator_yeartodate }
+
+Indicators can compute year-to-date values using the expression elements _yearToDate()_, _[periodInYear]_, and _[yearlyPeriodCount]_.
+
+In the examples below, #{a} can be: #{dataElementUID}, or any valid indicator expression item that returns a data value such as  #{dataElementUID.catOptionComboUid}, I{programIndicatorUID}, N{indicatorUID}, etc.
+
+| Indicator expression | Means |
+| --- | --- |
+| #{a} | current period value |
+| #{a}.yearToDate() | sum of values year to date. For example if the period is March gives the value for Jan+Feb+Mar  |
+| #{a}.yearToDate() / [periodInYear] | average year-to-date value. For example if the period is March gives the value for Jan+Feb+Mar / 3 |
+| #{a} - <nobr>#{a}.yearToDate()</nobr> / [periodInYear] | difference between current period and average year to date |
+| #{b} * [periodInYear] / [yearlyPeriodCount] | If #{b} represents the annual target population (for example, the number of people who should be vaccinated during this year), then this can show the number of people who should be vaccinated by the current period. For example in February this gives #{b} * 2 / 12.
```

Comment:
```suggestion
| #{b} * [periodInYear] / [yearlyPeriodCount] | If #{b} represents the annual target population (for example, the number of people who should be vaccinated during this year), then this can show the number of people who should be vaccinated by the current period. For example, in February this gives #{b} * 2 / 12.
```